landing_page_backend/contacts/management/commands/weekly_reports.py

```diff
@@ -0,0 +1,22 @@
+from django.core.mail import EmailMessage
+from django.core.management.base import BaseCommand
+
+from contacts.models import Contact
+from contacts.utils import write_data_to_sfv_file
+from landing_page_backend.settings import EMAIL_HOST_USER
+
+
+class Command(BaseCommand):
+    help = 'Weekly reports'
+
+    def handle(self, *args, **options):
+        contacts = Contact.objects.all().values_list('name', 'email', 'number', 'message')
+        for contact in contacts:
```

Comment:
I would move this cycle into the `write_data_to_csv_file` function.

That is an important point: 1 function should do 1 thing and do it well.
We deal here with writing to a file, so I'd like to have all the code for writing in a single place. Let's say, we'd like to write this data to another database later. That will make us sending 1 000 000 DB hits in case if we have 1 000 000 contacts. If all was done in a single function, we will just send a single request.